spec/services/atmosphere/appliance_vms_manager_spec.rb

```diff
@@ -115,7 +115,7 @@ def appliance(options)
 
     before do
       allow(vm_creator_class).to receive(:new).with(tmpl,
-        {flavor: flavor, name: name, user_data: 'user data', user_key: 'user key'})
+        {flavor: flavor, name: name, user_data: 'user data', user_key: 'user key', nic: nil})
```

Comment:
Line is too long. [93/80]<br>Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.